File: cpp/WORK/W7/W7_2/EBook.cpp
#include "EBook.h"

void EBook::showEBookInfo()
{
	cout << "Title: " << this->title << endl;
	cout << "Author: " << this->author << endl;
	cout << "Price: " << this->price << endl;
	cout << "Format: " << this->format << endl;	
}

File: csharp/ConsoleApp3/ConsoleApp3/EBook.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConsoleApp3
{
    public class EBook : Book
    {
        double FileSize { get; set; }

        public EBook(string title, string author, string isbn, double fileSize) :
            base(title, author, isbn)
        {
            FileSize = fileSize;
        }

        public override void DisplayInfo()
        {            
            base.DisplayInfo();
            Console.WriteLine($"File Size: {FileSize} KB");
        }
    }
}
